cromwell-backend/src/main/scala/cromwell/backend/model/TaskDescriptor.scala

```diff
@@ -0,0 +1,24 @@
+package cromwell.backend.model
+
+import wdl4s.TaskOutput
+import wdl4s.command.CommandPart
+import wdl4s.values.WdlValue
+
+/**
+  * Represents basic data needed to execute a task in a defined backend.
+ *
+  * @param name Task name.
+  * @param user User who want to execute the task.
+  * @param commandTemplate Arguments for the execution.
+  * @param workingDir Task working directory.
+  * @param outputs Outputs generated by the execution.
+  * @param runtimeAttributes Configuration needed at runtime in order to execute a task.
+  */
+case class TaskDescriptor(name: String,
+                          user: String,
+                          commandTemplate: Seq[CommandPart], //TODO: Should be Wdl Agnostic
```

Comment:
+1 to the tracking idea. Btw... Have you guys thought about using Jira as Cromwell's tracking system? I know you are using it internally and you may want to see this => https://www.atlassian.com/software/views/open-source-license-request/